extras/kms/docs.go

```diff
@@ -0,0 +1,79 @@
+// kms is a package that provides key management system features for
```

Comment:
I would be great if this is reviewed for context.

Reply:
LGTM!